ggu/main.c

```diff
@@ -345,7 +345,7 @@ submit_ggu(struct rte_mbuf **pkts, unsigned int num_pkts,
 	unsigned int i;
 	int ret;
 
-	RTE_VERIFY(num_pkts <= RTE_DIM(req->pkts));
+	RTE_VERIFY(num_pkts <= ggu_conf->gk->back_max_pkt_burst);
```

Comment:
As explained in another comment, `ggu_conf->gk->back_max_pkt_burst` should not be used here, but a mailbox parameter instead.